contrib/sys.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env drgn
+# Copyright (c) SUSE Linux.
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+"""Display system information and configuration data."""
+
+from datetime import datetime
+from datetime import timedelta
+
+from drgn.helpers.linux import for_each_online_cpu
+from drgn.helpers.linux.pid import for_each_task
+
+
+def print_line(key, value):
+    print(f"{key:<16} {value}")
+
+
+uts = prog["init_uts_ns"].name
+
+timekeeper = prog["shadow_timekeeper"]
+date = datetime.fromtimestamp(timekeeper.xtime_sec).strftime("%c")
+uptime = timedelta(seconds=timekeeper.ktime_sec.value_())
+avenrun = [f"{x.value_() / (1 << 11):.2f}" for x in prog["avenrun"]]
+totalram = (prog['PAGE_SIZE'] * prog['_totalram_pages'].counter).value_()
```

Comment:
If you want to support kernels before 5.0, you'll need to handle https://github.com/torvalds/linux/commit/ca79b0c211af63fa3276f0e3fd7dd9ada2439839:

```
$ python3 -m vmtest.vm -k '4.9.*' python3 -Bm drgn ./contrib/sys.py
...
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/home/osandov/repos/drgn2/drgn/__main__.py", line 16, in <module>
    main()
  File "/home/osandov/repos/drgn2/drgn/internal/cli.py", line 209, in main
    runpy.run_path(script, init_globals=init_globals, run_name="__main__")
  File "<frozen runpy>", line 291, in run_path
  File "<frozen runpy>", line 98, in _run_module_code
  File "<frozen runpy>", line 88, in _run_code
  File "./contrib/sys.py", line 25, in <module>
    totalram = (prog['PAGE_SIZE'] * prog['_totalram_pages'].counter).value_()
                                    ~~~~^^^^^^^^^^^^^^^^^^^
KeyError: '_totalram_pages'
```

As usual, in `contrib`, it's up to you if want to deal with that.

Reply:
I like that idea. There's a function in the kernel called `totalram_pages()`, so it makes sense to have the matching helper `drgn.helpers.linux.mm.totalram_pages()`.

Reply:
Fine, let's do that in #271.